services/movie.py

```diff
@@ -1,37 +1,44 @@
+from django.db import transaction
 from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
 
+import init_django_orm  # noqa: F401
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+        title: str = None
 ) -> QuerySet:
     queryset = Movie.objects.all()
 
     if genres_ids:
         queryset = queryset.filter(genres__id__in=genres_ids)
-
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
+    if title:
+        queryset = queryset.filter(title__icontains=title).values_list("title")
```

Comment:
Using `values_list('title')` will return a list of titles instead of `Movie` objects. If you need the full `Movie` objects, remove `.values_list('title')`.